### PR TITLE
[Python] Exclude relocating packages in homebrew

### LINKED ISSUE #198: Cleanup mac wheel
current for mac wheel we package way too many libs

### PR DESCRIPTION
**Pull request by eddyxu**
Closes #198 and #196

### CODE REVIEW

File: python/tools/delocate-wheel [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("--require-archs", metavar="ARCH")
+    parser.add_argument("-w", "--wheel-dir")
+    parser.add_argument("wheel", nargs="+")
+
+    args = parser.parse_args()
+    print(args)
```
Comment (changhiskhan) [inline python/tools/delocate-wheel:31]: Remove print?

File: python/tools/delocate-wheel [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    for wheel in args.wheel:
+        out_dir = args.wheel_dir
+        out_wheel = wheel
+        if out_dir:
+            if not os.path.exists(out_dir):
```
Comment (changhiskhan) [inline python/tools/delocate-wheel:36]: I don't think you need this check if exist_ok=True?

File: python/tools/delocate-wheel [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("--require-archs", metavar="ARCH")
+    parser.add_argument("-w", "--wheel-dir")
+    parser.add_argument("wheel", nargs="+")
+
+    args = parser.parse_args()
+    print(args)
+    for wheel in args.wheel:
+        out_dir = args.wheel_dir
```
Comment (changhiskhan) [inline python/tools/delocate-wheel:33]: What happens if out_dir is None? Should this be required?

Comment (eddyxu) [reply python/tools/delocate-wheel:28]: fixed.

**Review by changhiskhan — APPROVED** [review]
w00t

**Pull request merged**